cf-serverd/server_common.c

```diff
@@ -1003,20 +1003,10 @@ int StatFile(ServerConnectionState *conn, char *sendbuffer, char *ofilename)
 
         cfst.cf_readlink = linkbuf;
     }
-#endif /* !__MINGW32__ */
```

Comment:
islink = true;
can only be set within the `#ifndef __MINGW32__ ` block,, makes sense only on POSIX systems.
So, why have code checking that outside the block?